src/cmd/sync.rs

```diff
@@ -54,8 +56,18 @@ pub struct Args {
 
 #[tracing::instrument(level = "trace")]
 pub fn run(args: &Args) -> Result<()> {
-    let version = extract_version(args)?;
-    let repo = get_cros_dir_unchecked(&args.repo)?;
+    let is_arc = match (&args.cros, &args.arc) {
+        (Some(_), None) => false,
+        (None, Some(_)) => true,
+        _ => bail!("Please specify either --cros or --arc."),
+    };
+
+    let version = extract_version(args, &is_arc)?;
+    let repo = if is_arc {
+        get_cros_dir_unchecked(&args.arc)?
```

Comment:
I think it should be `get_arc_dir_unchecked`...

Reply:
I agree, but `get_arc_dir_unchecked()` does not exist. 
I'll add it in https://github.com/google/lium/pull/197 :)

Reply:
got it! then let's proceed!